presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcMetadataWriter.java

```diff
@@ -311,6 +312,10 @@ public int writeStripeFooter(SliceOutput output, StripeFooter footer)
 
     private static OrcProto.ColumnEncoding toColumnEncoding(ColumnEncoding columnEncodings)
     {
+        checkArgument(
+                !columnEncodings.getAdditionalSequenceEncodings().isPresent(),
+                "ORC writer doesn't support writing columns with non-zero sequence IDs: " + columnEncodings);
```

Comment:
nit:
```
Writing columns with non-zero sequence IDs is not supported in ORC: ... 
```

In Presto, ORC Writer usually mean the `ORCWriter` class that can write both ORC/DWRF. So the rewording will avoid the confusion :)